### ISSUE TITLE
Support passing args to gopls

### ISSUE THREAD

**Issue by luisdavim** [josa42/coc-go#64]
Hi, is there a way to pass args to the `gopls` command? I'm interested in passing `-remote=auto`
https://github.com/golang/tools/blob/master/gopls/doc/daemon.md

**Comment by josa42** [comment]
@luisdavim Currently there is no support for that.

**Comment by luisdavim** [comment]
I see this was added in master. Thanks.
Any plans for a release soon?

**Comment by josa42** [comment]
I just published `v0.7.0`. You should now be able to configure `gopls` args in your `coc-settings.json`:

```json
{
  "go.goplsArgs": ["-logfile=auto", "-rpc.trace", "-debug=localhost:6060"]
}
```
